src/main/scala/edu/berkeley/ce/rockslicing/Joint.scala

```diff
@@ -13,8 +13,8 @@ object Joint {
     * @param center The center of the joint plane. This should be in global coordinates.
     * @return
     */
-  private def findDistance(normalVec: Array[Double], localOrigin: Array[Double],
-                            center: Array[Double]): Double = {
+  def findDistance(normalVec: Array[Double], localOrigin: Array[Double],
+                           center: Array[Double]): Double = {
```

Comment:
Can you change the indentation here back to its original form? `center` should line up with `normalVec`.